.github/workflows/yocto-build-deploy.yml

```diff
@@ -673,8 +681,8 @@ jobs:
           RELEASE_VERSION: "${{ steps.balena-lib.outputs.os_version }}"
           BOOTABLE: 1
           TRANSLATION: "v6"
-          FINAL: ${{ steps.merge-test-result.outputs.finalize || inputs.finalize-hostapp }}
-          ESR: "${{ inputs.deploy-esr }}"
+          FINAL: ${{ steps.should-finalize.outputs.finalize }}
```

Comment:
Because this isn't a condition evaluated as true/false, I think this will just set an empty string.

Reply:
@klutchell it will yes, its used here: https://github.com/balena-os/balena-yocto-scripts/blob/33cf24f9db744c4ce822eef3a3488feec273f063/.github/workflows/yocto-build-deploy.yml#L796 and if its an empty string this if statement shouldn't be executed?

Reply:
ack, sounds good then!